draco/analysis/flagging.py

```diff
@@ -1199,37 +1202,25 @@ def mask_multi_channel(self, power, mask, times):
         # Get a median for each frequency
         med = weighted_median.weighted_median(power, (~mask).astype(power.dtype))
         # Set power to median when bright sources are in the sky
-        p = power.copy()
-        p[:, source_flag] = med[:, np.newaxis]
+        power[:, source_flag] = med[:, np.newaxis]
 
         # Subtract out a background, assuming that the type of
         # rfi we're looking for is very localised in frequency
-        p_med = weighted_median.moving_weighted_median(
-            p, (~mask).astype(p.dtype), size=(11, 3)
+        power -= weighted_median.moving_weighted_median(
```

Comment:
Yeah, this is one of the issues with this approach that I haven't been able to resolve. Even after applying the low-pass filter, there is some residual power from the galaxy and point sources appearing in sidelobes. I experimented with various window sizes, and I found that this residual power changes fast enough in frequency that, with larger windows, it would end up starting to mask some areas that were free of RFI. 

In the testing that I did, the only significant issue was the Russian satellite, since that's a larger blob of RFI. I had previously included the extra convolution with a gaussian kernel to try to fill in unmasked data around that, but I decided to remove that because the new sensitivity and chi-squared rfi masks are able to handle that sort of RFI better than this method.